verifier.py

```diff
@@ -141,7 +141,7 @@ def harvest_pages(self, json_list, url_end, page_class):
         for url in json_list:
             if url_end in url:
                 print('rel: ', url)
-                if url in run_info['error_list']:
+                if url in json_filename['error_list']:
```

Comment:
Isn't json_filename a string? This should be the dictionary created by opening the file with codecs.